code/_globalvars/logging.dm

```diff
@@ -10,6 +10,10 @@ GLOBAL_VAR(world_qdel_log)
 GLOBAL_PROTECT(world_qdel_log)
 GLOBAL_VAR(world_href_log)
 GLOBAL_PROTECT(world_href_log)
+GLOBAL_VAR(sql_error_log)
+GLOBAL_PROTECT(sql_error_log)
+GLOBAL_VAR(round_id)
+GLOBAL_PROTECT(round_id)
```

Comment:
If you protect `round_id`, how do you set it at `dbcore.dm#L79`

I had this problem when working on mine, and this breaks setting round ID

Reply:
GLOBAL_PROTECT pretty much just keeps things not supposed to be changeing it from changing it...Mainly varedit. So you know people can't varedit the roundtype to the word penis for kicks *looking at you neca*

Reply:
GLOBAL_PROTECT just shunts global variables outside of the standard GLOB list that admins can edit, into a protected one that is explicitly forbidden by can_vv_get.